SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -113,63 +128,68 @@ etc...
 
     > These lines ensure that the SpatialOS Unreal GDK can override Unreal's network replication.
     >
-    > If you ever need to run with Unreal networking for workflow or validation reasons, untick `Spatial Networking` in the `Play` drop down menu in the Unreal editor. This flag is maintained in your Unreal config files under `/Script/EngineSettings.GeneralProjectSettings` and is also valid for non-editor builds.
-    >
-    > You can easily switch between networking modes using this checkbox. As the SpatialOS Unreal GDK is in pre-alpha, switching back to Unreal's networking may be a viable way to work around some issues you may encounter, and may speed up your iteration. However, in Unreal networking mode you lose access to key multi-server features of the GDK, which may lead to erratic behavior.
+    > If you ever need to run with default Unreal networking for workflow or validation reasons, un-tick `Spatial Networking` in the `Play` drop down menu in the Unreal editor. This flag is maintained in your Unreal config file `<GameRoot>/Config/DefaultGame.ini` under `/Script/EngineSettings.GeneralProjectSettings` and is also valid for non-editor builds.
     >
-    > In the future, we expect to achieve parity with Unreal networking in terms of iteration speed and stability, thus removing the need for a networking mode switch.
-1. Open the game's uproject file in the Editor and navigate to *Edit > Project Settings... > SpatialOS Unreal GDK > Toolbar*.
+    > You can easily switch between networking modes using this checkbox. As the SpatialOS Unreal GDK is in alpha, switching back to Unreal's networking may be a viable way for debugging any issues you may be having and may speed up your iteration times. However, in Unreal default networking mode you lose access to the multi-server features of the GDK, which may lead to erratic behavior.
 
-1. Tag all replicated classes with the UCLASS tag `SpatialType`:
-    ``` 
-    UCLASS(SpatialType) 
-    ```
-1. Locate your game's `GameState`, and make it a public singleton:
+
+1. To provide context to SpatialOS on how to replicate [singleton classes]({{urlRoot}}/content/singleton-actors) (such as GameState and GameMode) we have added a `UCLASS` [specifier](https://docs.unrealengine.com/en-US/Programming/UnrealArchitecture/Reference/Classes/Specifiers) which provides said context. The next steps detail how to do this.
```

Comment:
* We call them Singleton Actors elsewhere, so should be consistent for clarity - unless there is a reason not to?
* I think we might need to qualify what these are a little more because this is likely to be the first time a user comes across the term. 
* The text has also slipped into "we".
* "said context" is awesome but not very doc-ish. 
* We don't need to say "next steps" but make the next steps indented to go under this one.
->
1. Specify Singleton Actors
The GDK uses [Singleton Actors]({{urlRoot}}/content/singleton-actors) - these are server-side authoritative Actors which are a single source of truth for both operations and data across a multi-server simulation; `GameState` and `GameMode` are examples of this. To provide context to SpatialOS on how to replicate Singleton Actors, the GDK  has a `UCLASS` [specifier (Unreal docs)](https://docs.unrealengine.com/en-US/Programming/UnrealArchitecture/Reference/Classes/Specifiers) to provide the context.

Reply:
Very nice. Added